### PR TITLE
Feat: notifications and log update

### PR DESCRIPTION
**Pull request by egor0798**
## Changes:
- Updated operation log window to show walles
- Updated notifications to show wallets

***

image
image

### CODE REVIEW

File: src/renderer/entities/notification/ui/NotificationRow/NotificationRow.tsx [hunk 1]
```diff
@@ -1,41 +1,37 @@
...
 import { useI18n } from '@renderer/app/providers';
-import { toAddress } from '@renderer/shared/lib/utils';
+import { WalletIcon, walletModel } from '@renderer/entities/wallet';
+import { Wallet } from '@renderer/shared/core';
 
 const NotificationBody = {
-  [MultisigNotificationType.ACCOUNT_INVITED]: (n: Notification, t: TFunction) => {
+  [MultisigNotificationType.ACCOUNT_INVITED]: (n: Notification, t: TFunction, w?: Wallet) => {
+    if (!w) {
+      return <div className="flex"></div>;
```
Comment (Asmadek) [inline src/renderer/entities/notification/ui/NotificationRow/NotificationRow.tsx:19]: what is the reason for this div? Why not simple fragment?

**Review by Asmadek — APPROVED** [review]

File: src/renderer/pages/Operations/components/Log.tsx [hunk 2]
```diff
@@ -36,6 +38,8 @@ const EventMessage: Partial<Record<SigningStatus | 'INITIATED', string>> = {
 const LogModal = ({ isOpen, onClose, tx, account, connection, contacts, accounts }: Props) => {
   const { t, dateLocale } = useI18n();
   const { getLiveTxEvents } = useMultisigEvent({});
+  const walletsMap = new Map(useUnit(walletModel.$wallets).map((w) => [w.id, w]));
+  const accountMap = new Map(useUnit(walletModel.$accounts).map((a) => [a.accountId, a]));
```
Comment (tuul-wq) [inline src/renderer/pages/Operations/components/Log.tsx:42]: `accountsMap` in plural

**Review by tuul-wq — APPROVED** [review]
With 1 comment

File: src/renderer/entities/notification/ui/NotificationRow/NotificationRow.tsx (hunk 1 above)
Comment (egor0798) [reply src/renderer/entities/notification/ui/NotificationRow/NotificationRow.tsx]: It was like that, but fragment is better so I replaced it with fragment

**Pull request merged**